test/powershell/Modules/Microsoft.PowerShell.Management/Set-Service.Tests.ps1

```diff
@@ -329,4 +329,17 @@ Describe "Set/New/Remove-Service cmdlet tests" -Tags "Feature", "RequireAdminOnW
         }
         { & $cmdlet @parameters } | Should -Throw -ErrorId $errorid
     }
+
+    It "Set-Service can run -Status Stopped to stop a service with dependencies" {
+        $servicename = "wuauserv"
+        $originalStatus = (Get-CimInstance -ClassName Win32_Service -Filter "Name='$($servicename)'").State
+        try {
+            Set-Service -Status Running $servicename
+            $script = { Set-Service -Status Stopped $servicename }
+            { & $script } | Should -Not -Throw
```

Comment:
The test should be povitive - if we are trying to stop a service we should check that the service stoped.
Perhaps print spooler service ("Spooler") stops/starts fasters.

Reply:
Thanks @iSazonov I will modify it.